Orange/evaluation/scoring.py

```diff
@@ -186,7 +186,10 @@ def compute_score(self, results, target=None):
         if n_classes < 2:
             raise ValueError("Class variable has less than two values")
         elif n_classes == 2:
-            return self.from_predicted(results, skl_metrics.roc_auc_score)
+            return np.fromiter(
```

Comment:
A test fail on travis, probably caused by this change. You can see the error returned by test here:

https://travis-ci.org/biolab/orange3/jobs/175538053#L4091